mpi4py: UCX failure at `MPI_Init_thread`

The following failure is not consistently reproducible, but it is the second time I see it in the last few days. 
Maybe it is a hiccup, but it could also be an actual issue related to recent changes.
Does it ring any bells to you?

https://github.com/mpi4py/mpi4py-testing/actions/runs/14849526249/job/41690426727#step:20:12
```
[1746494437.177960] [fv-az807-718:164991:0]         address.c:944  UCX  ERROR failed to unpack address, invalid bandwidth 0.00
Abort(406470671) on node 0: Fatal error in internal_Init_thread: Other MPI error, error stack:
internal_Init_thread(71)..............: MPI_Init_thread(argc=(nil), argv=(nil), required=3, provided=0x7ffcaf6e27e4) failed
MPII_Init_thread(283).................: 
MPIR_init_comm_world(35)..............: 
MPIR_Comm_commit(813).................: 
MPID_Comm_commit_pre_hook(145)........: 
MPIDI_UCX_mpi_comm_commit_pre_hook(19): 
MPIDI_UCX_comm_addr_exchange(117).....:  ucx function returned with failed status(ucx_init.c 117 MPIDI_UCX_comm_addr_exchange Invalid parameter)
[0]0:Return code = 15
[0]1:Return code = 0, signaled with Quit
Error: Process completed with exit code 15.
```

--- a/src/pm/util/pmiserv.h
+++ b/src/pm/util/pmiserv.h
@@ -58,7 +58,7 @@ typedef struct PMIGroup {
  * sockets and for sockets+shared memory can exceed 128 characters.
  */
 #define MAXKEYLEN    64 /* max length of key in keyval space */
-#define MAXVALLEN  4096 /* max length of value in keyval space */
+#define MAXVALLEN  1024 /* max length of value in keyval space */
 #define MAXNAMELEN  256 /* max length of various names */
 #define MAXKVSNAME  MAXNAMELEN  /* max length of a kvsname */
 typedef struct PMIKVPair {

